extensions/amp-apester-media/0.1/amp-apester-media.css

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* iframe */
+
+.iframe {
```

Comment:
@mrsufgi Apologies, I must have missed it before, but we can only use prefixed CSS names. 

- Use `amp-apester-media-` prefix for user-visible classes. E.g. `amp-apester-media-iframe`
- Use `-amp-apester-media-` prefixed for non-user-visible classes. E.g. `-amp-apester-media-iframe`.